api/config/v1alpha1/config_types.go

```diff
@@ -0,0 +1,25 @@
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+var _ runtime.Object = (*Config)(nil)
+
+// +kubebuilder:object:root=true
+
+// +genclient
+// +genclient:nonNamespaced
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// Config is the Schema for the Apoxy Backplane configuration.
+type Config struct {
```

Comment:
was this just for testing? I am guessing it was and it can be deleted before merge?